sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_utils.py

```diff
@@ -70,3 +70,27 @@ def parse_connection_string(connection_string: str) -> Tuple[str, str, str]:
 
 def get_current_utc_time() -> str:
     return str(datetime.utcnow().strftime("%b, %d %Y %H:%M:%S.%f ")) + "GMT"
+
+
+def get_key_filter(*args, **kwargs) -> Optional[str]:
+    key_filter = None
```

Comment:
Maybe we can put the logci

get_key_filter(*args, **kwargs) or kwargs.pop("key_filter", None),

into get_key_filter too?

Reply:
Updated